graphics/Coin/Portfile

```diff
@@ -1,80 +1,65 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
-PortGroup           qt4 1.0
-PortGroup           bitbucket 1.0
+PortSystem         1.0
```

Comment:
don't change the whitespace since it was correct in the previous one; according to the modeline it should be multiples of four spaces, so the righthand field should start at "21"

Reply:
Agreed.